consent-ui/src/app/ais/entry-page/entry-page.component.ts

```diff
@@ -7,8 +9,15 @@ import {Component, OnInit} from '@angular/core';
 })
 export class EntryPageComponent implements OnInit {
 
-  constructor() { }
+  private authid;
+  constructor(private route: ActivatedRoute, private cookieRenewalService: CookieRenewalService) { }
 
   ngOnInit() {
+    console.log('EntryPageComponent onInit');
+    this.route.paramMap.subscribe(p => {
+      this.authid = p.get('authId');
```

Comment:
use `p.authId` instead

Reply:
IDE complains: Property 'authId' does not exist on type ParamMap.